### DISCUSSION TITLE
TLS certificate not being privisioned

### DISCUSSION THREAD

**Discussion by Cynosure-North** [community/community#23863] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve got a custom domain for my page (cynosure.page), however github has been stuck here <img src="https://user-images.githubusercontent.com/52474045/181079752-735f2e84-725a-4787-937c-6372265c0dae.png" alt="image" data-base62-sha1="nNGz6hnZF9H1dTfLYXjccxvgO6P" width="547" height="119"> for ~24 hours. I have the domain configured to point to github </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/0/f014bd784a6dbe6ba453b9260b2e66a7e54e1cd3.png" data-download-href="/uploads/short-url/yfR1LbdCsylXhj8L58hmNf6h6v1.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/52474045/181079753-30e99030-2227-4391-856a-f06f2e9df824.png" alt="image" data-base62-sha1="yfR1LbdCsylXhj8L58hmNf6h6v1" width="690" height="226" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/0/f014bd784a6dbe6ba453b9260b2e66a7e54e1cd3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">748×245 17.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
, but it doesn’t seem to be working. The SSL certificate is only for github, and the domain has HSTS enabled, so I can’t connect. Am I doing something wrong?

**Comment by yoannchaudet** [comment]
<p>Hello and welcome here!</p>
<p>I am not sure why your certificate request was stuck in this state but I nudged it and it went through.</p>
<p>The certificate will take few minutes to be propagated through the CDN and you will be all good.</p>
